pkg/ui/workspaces/cluster-ui/src/util/format.ts

```diff
@@ -356,7 +356,7 @@ export function EncodeUriName(name: string): string {
   return encodeURIComponent(name).replace(/%25/g, "%252525");
 }
 
-export function EncodeDatabasesUri(db: string): string {
+function encodeDatabasesUri(db: string): string {
```

Comment:
Nice, i've noticed this a lot in our frontend code. Do you think we should check for this in our linter?

Reply:
Are you talking about the capitalization? Yeah, we could introduce a warning to lightly guide future var naming. I recall Andrii did some work to sync some linting rules between CC and db console so I'm surprised this doesn't exist already.